docker-for-azure/upgrade.md

```diff
@@ -42,6 +42,18 @@ $ docker run \
   docker4x/upgrade-azure:17.06.0-ce-azure1
 ```
 
+If you are already on a version more recent than 17.06.1 CE or 17.07.0 CE, you can use the below script to initiate the upgrade:
+
+```bash
+$ upgrade.sh YY.MM.X-ce-azureC
+```
+
+For example, to upgrade from 17.06.1 CE edge to 17.07.0 CE edge, you can use:
+
+```bash
+$ upgrade.sh 17.07.0-ce-azure1
```

Comment:
I think updating the  the latest version somewhere in release notes or this upgrade page would be ideal to keep users aware. I think having `upgrade.sh` from a past version try to compute any aspect of a future version will break if the logic it's relying on is not adhered to in the future version. For example, if a decision is made in the future to change the bucket `editions-us-east1` for any legitimate reason, the command line above will break.